pogobuf/lehmer.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
The only module I find which is close use constant different from the one used in pogo.
My code gives 100% compatibility and produce exactly the id from the actual app.